lib/modules/manager/cargo/extract.ts

```diff
@@ -184,11 +189,27 @@ export async function extractPackageFile(
       targetDeps = targetDeps.concat(deps);
     });
   }
+
+  const workspaceSection = cargoManifest.workspace;
+  let workspaceDeps: PackageDependency[] = [];
+  if (workspaceSection) {
+    const target = 'dependencies';
+    // workspace depedencies could potentially also be fully qualified
+    // like target.*.dependencies. It would require further unfolding of the sections
```

Comment:
I don't think so, there only the one defined[^1]
```suggestion
```

[^1]: https://doc.rust-lang.org/cargo/reference/workspaces.html#the-workspacedependencies-table